chunkflow/flow/setup_env.py

```diff
@@ -81,11 +81,10 @@ def setup_environment(dry_run, volume_start, volume_stop, volume_size, layer_pat
     cost = sys.float_info.max
     patch_num = None
     # patch number in x and y
-    is_isotropic = (voxel_size[0] == voxel_size[1] == voxel_size[2])
     factor = 2 ** mip
-    factor_z = factor if is_isotropic else 1
+    factor_z = factor if isotropic else 1
```

Comment:
It is a bit annoying the variable name changed from `is_isotropic` to `isotropic`, they are both OK for me but introduce a new variable in one commit and rename it in another one creates pointless noise in the git history.

Reply:
Maybe you can merge the renaming part to the first commit, so the second commit just promote the variable to an input parameters?

Reply:
Just merge the two commit into a single one is also an option, the second commit is not that big.